spec-dtslint/operators/mergeScan-spec.ts

```diff
@@ -21,6 +21,10 @@ it('should support a currency', () => {
   const o = of(1, 2, 3).pipe(mergeScan((acc, value) => of(acc + value), '', 47)); // $ExpectType Observable<string>
 });
 
+it('should support an index parameter', () => {
+  const o = of(1, 2, 3).pipe(mergeScan((acc, value, index) => of(index), 0)); // $ExpectType Observable<number>
+});
```

Comment:
this new test is unnecessary. :)

Reply:
I'm pretty sure this test will fail if the index is removed from the signature, so it guards against a regression, IMO.

Reply:
I removed this test. I was looking at how is this thing tested elsewhere and it seemed to be the same usecase as here https://github.com/ReactiveX/rxjs/blob/master/spec-dtslint/operators/map-spec.ts#L12-L14 or here https://github.com/ReactiveX/rxjs/blob/master/spec-dtslint/operators/findIndex-spec.ts#L8-L10

Reply:
Personally, I'd favour the dtslint typings tests being written without regard to what's tested in the specs - even if that means there are so excited redundant tests.